pkgs/tools/security/ronin/default.nix

```diff
@@ -12,8 +12,18 @@ bundlerApp {
     "ronin-repos"
     "ronin-vulns"
     "ronin-web"
+
+    # ronin-code-asm is a ruby library, which depends on ruby-yasm.
+    # ruby-yasm requires that yasm binary is in $PATH
+    "yasm"
   ];
 
+  buildInputs = [ pkgs.yasm ];
```

Comment:
So in most of the case, `buildInputs` should be empty or only `makeWrapper` which have to be used in `postBuild` script?

Reply:
My understanding of it is that because Nix can infer `buildInputs` from context, you should rarely need to set it explicitly. I really don't think this *must* change for this update but it was worth pointing out.